code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -3535,4 +3535,11 @@
 	taste_mult = 2
 	reagent_state = LIQUID
 	nutriment_factor = 40 //very filling
-	color = "#d169b2"
\ No newline at end of file
+	color = "#d169b2"
```

Comment:
See prior comment, except in Chemistry-Reagents-Food-Drinks_vr.dm

Reply:
that's not a change, that's whitespace fuckery :V

Reply:
oh nvm then